src/libraries/System.Text.Json/src/System/Text/Json/JsonEncodedText.cs

```diff
@@ -16,8 +16,8 @@ namespace System.Text.Json
     /// </remarks>
     public readonly struct JsonEncodedText : IEquatable<JsonEncodedText>
     {
-        private readonly byte[] _utf8Value;
-        private readonly string _value;
+        internal readonly byte[] _utf8Value;
```

Comment:
These could continue to be `private` no?

Reply:
Yes. Thanks. Forgot to undo earlier refactoring.